jupiter-tests/src/test/java/org/junit/jupiter/engine/execution/AbstractExecutableInvokerTests.java

```diff
@@ -98,6 +98,6 @@ private void register(ParameterResolver... resolvers) {
 
 	abstract void invokeMethod();
 
-	abstract <T> T invokeConstructor(Constructor<T> constructor, @Nullable Object outerInstance);
+	abstract <T> @Nullable T invokeConstructor(Constructor<T> constructor, @Nullable Object outerInstance);
```

Comment:
It does not sound right for `invokeConstructor` to return null. Does it mean the null signals the failure?